tao/tao_bridge/passes/tao_clone_constants_for_better_clustering.cc

```diff
@@ -93,6 +93,26 @@ StatusOr<bool> IsSmallHostConstant(Node* n) {
   return total_elements < kSmallTensorThreshold;
 }
 
+// almost same with IsSmallHostConstant but skip the check of op's size
+// for fakequant's scale and zero_point
+StatusOr<bool> IsFakeQuantArgumentConstant(Node* n) {
+  if (!n->IsConstant()) return false;
+
+  DeviceNameUtils::ParsedName parsed;
+  TF_RET_CHECK(
+      DeviceNameUtils::ParseFullName(n->assigned_device_name(), &parsed));
+  if (parsed.type != DEVICE_CPU) return false;
+
+  // multi DiscFakeQuant use one Const
+  if (n->type_string() == "Const" &&
+      std::all_of(n->out_nodes().begin(), n->out_nodes().end(), [](Node* out) {
```

Comment:
It's not necessary to require all the outputs to be DiscFakeQuant. maybe just check there are DiscFakeQuant user and the number of consumers of const is larger than 1?

Reply:
fixed